src/services/tx/txMonitor.ts

```diff
@@ -83,11 +83,12 @@ export const waitForRelayedTx = (taskId: string, txId: string): void => {
         }
 
         return res.json().then((data) => {
-          throw new Error(`${res.status} - ${res.statusText}: ${data?.error?.message}`)
+          throw new Error(data.message)
```

Comment:
Why change this?

Reply:
I adjusted it based on the error's JSON
![Screenshot 2023-03-27 at 17 24 15](https://user-images.githubusercontent.com/32431609/227987311-96c57da6-fdb8-4e50-95b2-af3788d1553d.png)